### ISSUE TITLE
How do I choose the TropChem simulation type and the 47 levels for a C48 simulation with GCHP v14.2.2?

### ISSUE THREAD

**Issue by herizonarivelo** [geoschem/gchp#364]
Name: Herizo Narivelo
Institution: Observatoire Midi-Pyrénées - Laboratoire d'aérologie (LA UMR 5560) - Université Toulouse III Paul Sabatier

Dear all, 

I am a newcomer to using the GCHP model. In my project, I am going to run a long simulation (multi-year) but before doing these extended simulations, I first tested a one-month simulation with c48 horizontal resolution on our local cluster. For information, I installed/used the GCHP v14.2.2 model and ran it on our local cluster, which does not have enough computing resources. In the previous version, for example, the GCHP v12.3.2, we could use the TropChem simulation type with 47 vertical levels and the UCX flux from the stratosphere only. However, in the case of GCHP v14.2.2, I cannot specify a TropChem but rather fullchem and by default 72 vertical levels when creating my run directory.  I would like to ask you if it will be possible to use a TropChem and 47 vertical levels with GCHP v14.2.2 and only the stratosphere UCX flux to optimise my simulations as we do not have many ressources in terms of processors and nodes. If this is the case, how can I achieve this? 
Thanks in advance for your advice.

Herizo Narivelo

**Comment by yantosca** [comment]
Thanks for writing @herizonarivelo.  GCHP cannot use the reduced 47L grid, as the MAPL I/O software that GCHP uses cannot do vertical regridding.   All GCHP simulations have to be done at 72L.

You can try to save memory by reducing the number of diagnostic outputs that are saved to disk.  

You also might explore using GCHP on the AWS cloud computing environment.  Tagging @yidant.

**Comment by sdeastham** [comment]
Agreeing with @yantosca - if you do want to run with only 47 layers you would need to switch the internal parameters in GCHP to 47 layers and manually regrid the input meteorology. This would be a significant undertaking but is in theory doable - we have run GCHP with 26 layers using CAM input, for example. There are plans for vertical regridding to be performed online in GCHP but there is no timeline for that change as yet.

**Comment by lizziel** [comment]
Hi @herizonarivelo, since you will be running at C48 you could also consider using GC-Classic at 2x2.5. This is the roughly equivalent resolution on lat-lon grid and would give you the option of running on a 47L vertical grid.

**Comment by herizonarivelo** [comment]
Hi everyone, thank you very much for your advice @yantosca, I will try to reduce the number of diagnostic outputs. Thanks also to you @lizziel , I did a one month fullchem type simulation on Geos-Chem Classic with 47L and 2x2.5 horizontal resolution but with only one node and 16 CPUs, my one month simulation took a lot of time, that's why I am going to use GCHP to be able to use several nodes for my multi-year simulations. Thanks to others for their advice. I will come back to you if I have any questions. Thanks again.

**Comment by yantosca** [comment]
Closing out this issue. Feel free to open a new issue if you are stuck.

**Comment by herizonarivelo** [comment]
Dear All,
I am back to you to ask another question. 
My project is to work on a long simulation ( several years) but before doing this simulation, I am doing a simulation test in order to choose the start date of the simulation. To do this, I have tried not enabling chemistry, but I seem to be taking a long time with my simulation. For one month period of simulation, it has taken approximately 1 hour of time to complete the run. 
Some time ago, a simulation carried out with the previous version of Geos-chem took a few hours to complete simulation over a long period.
As I mentionned above, I used the GCHP v14.2.2 version for my study. For the simulation with only emission, I had set false the fields Chemistry ; advection, Cloud, PBL_Mixing, Non_Local_Mixing, DryDep and WetDep in the configuration file setCommonRunSettings.sh and also set photolysis to false in geoschem_config.yml. 

Is there anything I should set to false/comment in a configuration file other than the one I have set to false in setCommonRunSettings.sh? 
[gchp.20070701_0000z.log](https://github.com/geoschem/GCHP/files/13871333/gchp.20070701_0000z.log)
[setCommonRunSettings.sh.log](https://github.com/geoschem/GCHP/files/13871351/setCommonRunSettings.sh.log)

Thanks in advance for your returns.
